### ISSUE TITLE
PDFUA-62: 1.3.1 Info and Relationships | Structure element: Figure | non-compliant

### ISSUE THREAD

**Issue by zakkinsey** [zakkinsey/pdfa-test-4#62]
Jira issue originally created by user @:

Figure with text, where text was pasted in "Actual text" (figure is not shown in screen reader)

**Comment by zakkinsey** [comment]
Comment created by @:

What does mean: not shown in screen reader? The purpose of ActualText is presenting text, that is not technically text. From this point of view it is correct to not present the picture. If this is intended, ALT text should be used.

Remark: The original example is not well tagged, as <Figure> should be child go a semantic correct tag, in this case <P> – example file added

**Comment by zakkinsey** [comment]
Comment created by @mrbhardy:

Insufficient example because there isn't sufficient information to show whether this is correct.
